internal/types/types.go

```diff
@@ -131,3 +131,36 @@ func (ve *ValidationError) IsError() bool {
 func (ve *ValidationError) IsWarning() bool {
 	return ve.Level == Warning
 }
+
+// NewDefaultConfig initializes the configuration with default values for testing.
+func (c *Config) NewDefaultConfig() {
```

Comment:
1. If that's for testing, it should be in `_test.go` file.
2. There is no need to set default values (like empty strings, `false` for booleans etc).
3. The name `Default` is misleading -- there is no "default" config.

Perhaps we can have a testing function named `getEmptyConfig` or somesuch.

Or, fix the code that uses the config to allow a truly default config (i.e. `cfg := types.Config{}` should work). If you are going to do that, please do it in a separate commit.

Reply:
Seem like reasonable changes, but maybe for a separate PR now?